examples/iotdashboard/nodes/RadialTickmarksNode.h

```diff
@@ -0,0 +1,27 @@
+/******************************************************************************
+ * QSkinny - Copyright (C) 2016 Uwe Rathmann
+ * This file may be used under the terms of the 3-clause BSD License
+ *****************************************************************************/
+
+#pragma once
+
+#include <QskArcMetrics.h>
+#include <QskIntervalF.h>
+#include <QskScaleTickmarks.h>
+
+#include <QSGGeometryNode>
+
+class RadialTickmarksNodePrivate;
+
+class RadialTickmarksNode : public QSGGeometryNode
```

Comment:
This class shares a lot of code with QskTickmarksNode; once this PR is merged, we could unify them into an abstract class QskTickmarksNode and two child classes QskLinearTickmarksNode and QskRadialTickmarksNode or so.